src/languageservice/services/yamlCompletion.ts

```diff
@@ -535,7 +535,7 @@ export class YAMLCompletion extends JSONCompletion {
           filterText = insertText.replace(/[\n]/g, ''); // remove new lines
         }
         collector.add({
-          kind: this.getSuggestionKind(type),
+          kind: s.suggestionKind || this.getSuggestionKind(type),
```

Comment:
Is this change needed for this PR? If so I think we should have a simple test case covering it just to make sure everything works

Reply:
I'm using it with custom array snippets where I need to set custom symbol.
I added UT for this case.

If you think that custom suggestionKind shouldn't be there. I can remove it and add it locally.

Reply:
I think you can keep it in